src/Tizen.NUI/src/public/BaseComponents/Style/ViewStyleBindableProperty.cs

```diff
@@ -346,9 +345,8 @@ public partial class ViewStyle
                 if (viewStyle.size == null)
                 {
                     if ((float)newValue == 0) return;
-                    viewStyle.size = new Size();
                 }
-                viewStyle.size.Height = (float)newValue;
+                viewStyle.size = new Size(viewStyle.size?.Width ?? 0, (float)newValue);
```

Comment:
very good to use ?? operator
(https://docs.microsoft.com/ko-kr/dotnet/csharp/language-reference/operators/null-coalescing-operator)

Reply:
Thank you.